### DISCUSSION TITLE
关于no_transaction for PR#7401

### DISCUSSION THREAD

**Discussion by VersaillesHeroes** [consenlabs/webview#64] [Q&A]
I pull request #7401 on 16 Sept. Your team member commented it "no_transaction". I fixed it on 19 Sept.

I have one latest token transaction in the last 10 days. Please refer to: https://etherscan.io/tx/0xc961f8875a610ed0cd581438aee2de7fd1254fc24915ddf85a79dfc071e71c36

However, my PR#7401 is still open and no further actions from your side. 
@imtoken-jeffrey @beenhero @kaichen @nuaays

**Comment by xwartz** [comment]
I don't understand what you mean, is it a question in the wrong place?
